### PR TITLE
[Reviewer: Andy] Add prompt to delete_user

### PR DESCRIPTION
**Pull request by Tom-Rees**
Hi Andy,

This is an upstream merge of a fix to add a prompt to the command line 'delete_user' script. I've tested it live on our test system.

Please could you review?

Cheers,
Tom

### CODE REVIEW

File: src/metaswitch/ellis/prov_tools/delete_user.py [hunk 1]
```diff
@@ -57,6 +58,17 @@ def main():
...
 
+    if not args.no_prompt:
+        print 'Are you sure you wish to delete the following users: {}? [y/n]'.format(args.dns)
+        choice = raw_input().lower()
+
+        if choice != 'y':
+            if choice != 'n':
+                print "Invalid response. Aborting"
+            else:
+                print "Aborting"
```
Comment (bossmc) [inline src/metaswitch/ellis/prov_tools/delete_user.py]: "Aborting" sounds a little aggressive, generally CLI tools go for "Exiting on user request" or similar so the user's action gets direct feedback.

**Comment by bossmc** [comment]
One user experience comment inline, otherwise fine.

**Comment by Tom-Rees** [comment]
Changes made.

I wasn't sure whether to pass this back for review or just merge it so I thought I'd err on the safe side. Please let me know if I should be merging after review feedback in future.

Cheers,
Tom

**Comment by bossmc** [comment]
Looks good.  Generally if the markups are non-controversial, you can just merge yourself.  It's a bit of a judgement call.

**Pull request merged**